roles/common/tasks/main.yml

```diff
@@ -1,53 +0,0 @@
-- name: Set FQDN hostname
```

Comment:
Split this one in two: 

- `configure-apt`
- `common-server` that depends on many roles, and requires having apt configured.